content/en/docs/concepts/configuration/configmap.md

```diff
@@ -279,5 +279,6 @@ to the deleted ConfigMap, it is recommended to recreate these pods.
 
 * Read about [Secrets](/docs/concepts/configuration/secret/).
 * Read [Configure a Pod to Use a ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/).
+* Read about [changing a ConfigMap (or any other Kubernetes object)] (/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch/)
```

Comment:
To make the link work:

```suggestion
* Read about [changing a ConfigMap (or any other Kubernetes object)](/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch/)
```

Reply:
Wonderful. Thank you and apologies. Will try with a new PR since this is close.

Reply:
@ProbStub : I do not see the requirement for a change in the first place. The existing link and content seem okay to me. I don't think you need to make amendments with another PR.

Reply:
Your call. I spent 2h to find a way how to update existing objects. If there is information on how to create and use ConfigMap perhaps users should also know how to change ConfigMaps.

Reply:
@ProbStub To clarify what I meant, if you're adding a new link you don't need to open a new PR. You can reopen this one.

Reply:
@divya-mohan0209 I see. Thank you for reopening :)

Reply:
Apologies for the unintentional miscommunication.

Reply:
No worries. It is my first contribution here so am happy for the detailed review. Thanks for all the feedback!